transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/util/ImportExportPackageUpdater.java

```diff
@@ -199,7 +269,7 @@ private String addELPackage(String importPkgs) {
 			importPkgs = "";
 		}
 		if (!importPkgs.isEmpty()){
-			importPkgs += ",\n"; //$NON-NLS-1$
+			importPkgs += ",\n "; //$NON-NLS-1$

```

Comment:
should we care about max line widt / max chars per statement in the manifest and do the appropriate line break if needed?